apps/chronicles/types.tsx

```diff
@@ -17,6 +17,8 @@ export type DiscoverStackParamList = {
     Discover: undefined;
     Paper: undefined;
     Button: undefined;
+    Switch: undefined;
+    SmallSwitch: undefined;
```

Comment:
We don't need a separate navigation route for the small switch since it is shown on the same screen as the big one